src/main/java/com/notably/logic/parser/DeleteCommandParser.java

```diff
@@ -36,17 +40,18 @@ public DeleteCommandParser(Model notablyModel, CorrectionEngine<AbsolutePath> pa
         if (!ParserUtil.arePrefixesPresent(argMultimap, PREFIX_TITLE)
                 || !argMultimap.getPreamble().isEmpty()) {
             title = args.trim();
-            if (title.isEmpty()) {
-                throw new ParseException("Path cannot be empty");
-            }
         } else {
             title = argMultimap.getValue(PREFIX_TITLE).get();
         }
 
+        if (title.isEmpty()) {
```

Comment:
Actually, since we don't allow whitespace for title, we should use `isBlank()` here. This is because `<whitespace>` is considered not empty, but it is blank.

https://stackoverflow.com/questions/23419087/stringutils-isblank-vs-string-isempty

Reply:
I agree with Johanna, since that makes your intention more explicit. However, technically speaking, using `title.isEmpty()` doesn't matter here! The title's content would have been trimmed by the time it reaches here.